docs/operator-manual/applicationset/Controlling-Resource-Modification.md

```diff
@@ -77,6 +77,22 @@ spec:
     applicationsSync: create-update
 ```
 
+### How to prevent Application controller from deleting Applications when deleting ApplicationSet
```

Comment:
The policy descriptions above at lines 42 - 63 state that the policy setting prevents modifications/deletion of applications, and we are stating here that the policy setting is not effective to prevent the deletion. I think this is causing some inconsistency in the documentation. 

Can we add a line in above sections for pointing to this section or incorporate this section in each of the above sections? WDYT?

Reply:
Added a line for pointing to this section.